docs/arc42/08_concepts.md

```diff
@@ -20,6 +20,20 @@ the respective material number of a partner leading to the following constellati
     - own material number = material number supplier
     - material number partner = material number customer
 
+Furthermore, as of ItemStockSAMM v2.0.0 the supplier CX ID is the sole identifier for a material of a respective partner.
+This leads to two scenarios, within the shared asset approach:
+
+- when acting as a customer, the supplier's global asset id (Catena-X ID) is leading and taken from the PartTypeInformation of the supplier's twin.
+- when acting as a supplier, the initially generated global asset id (Catena-X ID) is handed over to the customers.
+
+Within PURIS this results in the following steps:
+
+- when creating a material (independent of the role) a global asset id (Catena-X ID) is defined on the material, which is used for the supplier twin.
+- when creating a mpr, the global asset id (Catena-X ID)
+    - may be set externally (controller),
+    - is taken over from the suppliers PartTypeInformation (acting as supplier) **OR**
+    - is set to the material's catena-X id (acting as customer)
```

Comment:
No, the third case is wrong. The Material's CX Id is never set as the partner CX Id in any case. 
The partner CX Id is always only set to the CX id, that the partner has defined. 
Typically, on creation of an MPR, the partnerCX Id should be left blank ( i.e "null"). The puris backend will then automatically fetch the partner CX id via the PartTypeInformation endpoint of that partner, __if__ the partner has a supplier flag. 
If this MPR's "partnerSupplies" flag is set to false initially, the partnerCXId field of the MPR remains "null". 
If this MPR's "partnerSupplies" flag is set to true at a later point in time, then that partners CX Id will also be fetched automatically via the PartTypeInformation endpoint of that partner. 

And the first case, i.e. the option to manually set the partnerCXId (for example via the put or post methods of the materialpartnerrelations controller should only be used in exceptional cases, when -for some unforeseen reason - the fetch from the partner's PartTypeInformation api does not work.  I.e. normally that should never happen.